src/main/java/br/com/caelum/comidinhas/handler/validation/FieldErrorOutputDto.java

```diff
@@ -0,0 +1,22 @@
+package br.com.caelum.comidinhas.handler.validation;
+
+public class FieldErrorOutputDto {
+
+    private String field;
+    private String message;
```

Comment:
A classe não precisa ser pública e os atributos podem ser final.

```suggestion
class FieldErrorOutputDto {

    private final String field;
    private final String message;
```